lib/matplotlib/axes/_axes.py

```diff
@@ -4617,6 +4617,15 @@ def _parse_scatter_color_args(c, edgecolors, kwargs, xsize,
         if edgecolors is None and not mpl.rcParams['_internal.classic_mode']:
             edgecolors = mpl.rcParams['scatter.edgecolors']
 
+        # Raise a warning if both `c` and `facecolor` are set (issue #24404).
+        if c is not None and facecolors is not None:
+            _api.warn_external(
+                "You passed both c and facecolor/facecolors for the markers. "
+                "Matplotlib is ignoring the facecolor "
+                "in favor of what you specified in c. "
```

Comment:
Slightly more concise:
```suggestion
                "Passing both c and facecolor/facecolors for the markers "
                "is ambiguous. facecolor is ignored in in favor of c. "
```

Reply:
Thank you for reviewing the comment. Your change is more concise, and I understand the argument that matplotlib developers consider passing both as ambiguous. 
However, a user intentionally passing both will likely not understand why it is ambiguous. They probably assumed incorrectly, as the issue's Original Poster did, that c defines both a face color and edge color and that just like adding edge color works, the face color can also be overwritten.